store/store.go

```diff
@@ -441,15 +439,11 @@ func (s *store) Prune(resultFunc func(pruned, skipped, pruningHeight uint32)) er
 			return err
 		}
 
-		if deleted {
-			resultFunc(1, 0, i)
-
-			continue
-		}
-
-		resultFunc(0, 1, i)
+		resultFunc(deleted, i)
 	}
 
+	s.isPruned = true
```

Comment:
This value stores in memory after prune needs to restart node.

Reply:
Done.